datameta/static/js/admin.js

```diff
@@ -179,16 +179,36 @@ DataMeta.admin.reload = function() {
         })
         .then(response => response.json())
         .then(function (json) {
+
             DataMeta.admin.reload_requests(json.reg_requests, json.groups);
             DataMeta.admin.subnav();
             DataMeta.admin.rebuildUserTable(json.users);
-            DataMeta.admin.rebuildGroupTable(json.groups);            })
+            
+            fetch(DataMeta.api('users/self'),
+            {
+                method: 'GET'
+            })
+            .then(response => response.json())
+            .then(function (self_json) {
+                if(self_json.siteAdmin) {
+                    document.getElementById("nav-groups-tab-li").style.display = "";
+                    document.getElementById("nav-metadata-tab-li").style.display = "";
+                    document.getElementById("nav-site-tab-li").style.display = "";
+                    DataMeta.admin.getAppSettings();
+                    DataMeta.admin.getMetadata();
+                    DataMeta.admin.rebuildGroupTable(json.groups);  
+                }
+            })
+            .catch((error) => {
+                console.log(error);
+            });
```

Comment:
Shall we move this into `datameta.js` and execute it on every page load, say store the response json in `DataMeta.user`? That way it would always be available in the application context, I think we'll need this more often.